tendrl/node_agent/message/handler.py

```diff
@@ -57,7 +58,13 @@ def bind_unix_listener(self):
             self.sock.setblocking(0)
             self.sock.bind(socket_path)
             self.sock.listen(50)
-        except (TypeError, BlockingIOError, socket_error, ValueError) as ex:
+            gid = pwd.getpwnam("tendrl-user").pw_gid
```

Comment:
When 'tendrl-user' is created? and who creates it
Is it a good idea to assign permissions for all the users in a group rather than doing it on per user basis? @brainfunked , @r0h4n